<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>766</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1043</width>
    <height>766</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1043</width>
    <height>766</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>答题卡判卷</string>
  </property>
  <widget class="QPushButton" name="btn_ansload">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>670</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_correct">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>670</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>批改缓存</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>961</width>
     <height>631</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="img_origin">
      <property name="text">
       <string>未处理的答题卡</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="img_new">
      <property name="text">
       <string>批改后答题卡</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
